httomo/task_runner.py

```diff
@@ -266,17 +300,14 @@ def run_tasks(
             method_func.parameters.update({"ncore": ncore})
 
         idx += 1
-        reslice_info, glob_stats[idx] = run_method(
+        glob_stats[idx] = run_method(
```

Comment:
we don't really need `glob_stats` that we get from `run_method`, do we?. So, run method can just _run_ method, not return anything.

Reply:
It's a good question, I'm not sure to be honest.

I think stats from each method can be useful info to provide at the end of the run or something, which is maybe why Daniil asked me to put `glob_stats` in. When he's back we can check if we need the stats info from a method execution to be kept afterwards or not!

Reply:
I think by default `glob_stats` are associated with every method result as other methods in the pipeline might request it. However, with the blocks loop this makes it a bit tricky...

It seems that the stats should be calculated after the loop over blocks is finished, so only in the sections loop, where the data on the cpu is available as a whole. 

This is not ideal of course... As for global_stats in image saver we should be probably OK as it is the CPU section, but for stats that are needed, say, in the median filter we cannot much to offer except `local_stats` which is not a great idea as the data in blocks will be treated differently. So I think we should be aware of that... 

Could we for now remove `glob_stats` from `run_method` but rather add it to the end of the blocks loop?

Reply:
Yeah, I see your point that the blocks loop kinda gets in the way of calculating stats on the result of each method.

Sure, I'm happy to make that change. Just to make sure I've understood this correctly, does this mean that stats will only be calculated on the results of methods that are at _the end of sections_, since we'd be waiting for all blocks to complete in a section before doing stats stuff?

Reply:
yes, it seems so that stats will be **only** available for the last method in the section. 

May be there is more torturous way to make it work for every method by calculating stats on cupy arrays for every block and then collect it afterwards, but for now, I don't think that we should do it. Crucially we need stats while saving images and this is a new section, so we can get it calculated for it. 

So I guess we should change the terminology to `global_stats_section` or something? And may be it will more stored in the meta data of the section?

Reply:
Hmm, let's see... Currently, `glob_stats` is a dict associating stats with dataset names. If we want the option in the future  to associate stats to the result of every method, then it's probably best to keep that with the dict structure it currently has, and not rename it to `glob_stats_section`?

For the stats of a section, _this_ would make sense to be in the `PlatformSection` dataclass, we can call it something like `output_stats` or something, and then we use this as the way to easily pass stats into the image saver for now?

Reply:
sounds good, thanks.